etc/adminhtml/system.xml

```diff
@@ -5,12 +5,25 @@
         <section id="system" translate="label" type="text" sortOrder="900" showInDefault="1">
             <tab>advanced</tab>
             <resource>Aligent_AsyncEvents::manage</resource>
-            <group id="async_events" translate="label" type="text" sortOrder="1000" showInDefault="1">
+            <group id="async_events" translate="label" type="text" sortOrder="1000" showInDefault="1" showInStore="1" showInWebsite="1">
                 <label>Async Events</label>
-                <field id="max_deaths" translate="label" type="text" sortOrder="10" showInDefault="1">
+                <field id="max_deaths" translate="label" type="text" sortOrder="10" showInDefault="1" showInStore="1" showInWebsite="1">
                     <label>Maximum Deaths</label>
                     <validate>integer validate-not-negative-number</validate>
                 </field>
+                <field id="subscriber_log_cleanup_cron" translate="label" type="select" sortOrder="20" showInDefault="1" showInStore="1" showInWebsite="1">
+                    <label>Subscriber Log Cleanup</label>
+                    <comment>Enable or disable the subscriber log cleanup cron. </comment>
+                    <source_model>Magento\Config\Model\Config\Source\Enabledisable</source_model>
+                </field>
+                <field id="subscriber_log_cron_delete_period" translate="label" type="text" sortOrder="25" showInDefault="1" showInStore="1" showInWebsite="1">
+                    <depends>
+                        <field id="system/async_events/subscriber_log_cleanup_cron">1</field>
+                    </depends>
+                    <label>Oldest Subscriber Log (days)</label>
```

Comment:
Maybe reword this to something like Asynchronous Event Log Lifetime (Days) or something similar.